SPECS/glusterfs/glusterfs.spec

```diff
@@ -1,16 +1,27 @@
+#
+# Use mariner=<version> for Mariner-specific settings
+#
+%global mariner 3
```

Comment:
Not needed, just remove the `fedora` and `rhel` specs as if `fedora == 32`. If you got this update from a newer version of Fedora than 32, take that number (and mentione the Fedora version in the changelog).

Reply:
i find this variable an easy-to-understand way of communicating choices explicitly with the next upgrader.

Reply:
removed